+++
title = "Genes affecting human intelligence"

+++
Most people know that there is something tangible called intelligence.
Ancient Hindus saw it as a tangible entity called buddhi, while modern
psychometrics uses the concept of general intelligence called g. In the
modern theory it is proposed that irrespective of the domain of
specialization, and despite the presence of general abilities such as
verbal ability, or spatial ability or numerical ability that contribute
to intelligence, the main common determinant of intelligence is this
generic factor termed g. IQ tests are designed to be as good a proxy for
this g as possible. Generally this theory of g also makes intuitive
sense and fits in with our anecdotal observations: Most scientists of
the type who need to use their mental faculties for their profession
have IQ higher than 135. There is normal distribution of IQ, which fits
well with the anecdotal observation that the majority of people in the
population fall around a certain average intelligence, a yard-stick used
for educational testing. There are few highly intelligent people in any
population. g is a quantitative trait with broad sense heritability
H^2\~0.5-0.8 showing that it is substantially genetically determined. It
is correlated with brain anatomy like frontal lobe gray matter volume
and brain physiology like glucose utilization. Thus, the available
evidence strongly supports intelligence as a biologically determined
trait with a strong genetic component.

So what are the genes that affect human intelligence? As I was studying
brain function genes in context of the pathogen-response of mammalian
hosts to a well-known vertebrate parasite, I was distracted to survey
intelligence related genes. I gathered a bit of biology in this regard
mainly for my own edification, but in course of a conversation with R
decided that it might be worthwhile putting down some of it here in the
near future.

A brief summary:  
CHRM2: The cholinergic muscarinic 2 receptor gene encodes a 7 TM
receptor of acetylcholine expressed in the brain (and heart). This gene
shows a highly significant association with intelligence with the
strongest association seen with the SNP: rs324650. The T allele (as
against the ancestral A allele) of this gene was found to correlate with
an increase in what the authors call performance IQ by about 4.6 points.
This SNP is seen in the 3’UTR intron suggesting that its biological
action is regulatory, probably at the transcriptional level. In studied
human populations the T allele shows about 91% frequency in Chinese and
Japanese, 47% in White Americans, and 27% in West Africans.

DTNBP1: Several SNPs in this gene were associated with IQ. This gene
encodes a cytoskeletal coiled-coil protein that affects dystrobrevin
localization in axons and might thereby affect axon architecture.
rs760761, rs2619522, rs2619538 are the 3 main SNPs that affect what the
authors term full-scale IQ. The first 2 derived SNPs reduce IQ while the
3rd one increases IQ, each by about 6-7 points. Again these seem to
cause regulatory effects as they are found in non-coding intronic
regions. Each allele shows clear population differences: For example in
the last SNP the T allele is similar in proportion in both West Africans
and White Americans, but is extremely rare in the East Asians.

SNAP25: This gene encodes a protein involved in vesicular fusion, with
two coiled coil SNARE modules and cysteines which are palmitoylated. 4
SNPs in the intron-1 of this gene: rs363043, rs353016, rs363039 and
rs363050 have an effect of IQ. These SNPs appear to alter transcription
factor binding sites and alter the expression pattern of the SNAP-25
gene. Some of these alleles again show dramatic population differences.

NRG1: Neuregulin-1 is a cell-surface protein which is the ligand for the
ERBB3 and ERBB4 receptor tyrosine kinases. It induces the expression of
acetylcholine receptor and induces Schwann cell proliferation. One
variant of it in the promoter region showed reduced IQ and
frontal/temporal lobe activity and pre-disposition for pyschosis.  
  
LIMK1: The LIM domain kinase 1 with two LIM domains fused to a kinase
domain, has been implicated in William’s syndrome. Effects on
visuo-spatial cognition and the tendency of individuals with LIMK1
deletion to anthropomorphize non-human entities implicate it in both
spatial ability and general reconstruction of imagery. Mice with LIMK1
deletion also show altered spatial abilities and fear responses. Its
potential interaction with the cytoplasmic tail of NRG1 implicate it in
a common pathway with that gene product. This might be explored in the
future for a role in intelligence.

RIMS1: Encodes a protein with N-terminal Zn-chelating Rabphilin-effector
domain fused to a C-terminal PDZ, and 2 C2 domains. Appears to be
important in regulating synaptic membrane exocytosis via the Rab3 GTPase
during neurotransmitter release. Importantly, it physically interacts
with SNAP-25 which has also been implicated in intelligence and
vesicular fusion. Here a mutation in the coding region, resulting in a
R844H (gi:2224621) substitution, in turn results in increased IQ,
especially verbal IQ \[ekanetra, I wonder if you have this mutation\!\].
This mutation is in the C2 domain and is close to the residues
interacting directly with Calcium. We believe that the enhanced RIMS1 IQ
phenotype might be linked to altered calcium affinity of the molecule.
The down side of this IQ gain is an associated vision defect phenotype.
Mice lacking this gene show severely impaired learning and memory  
  
COMT: catechol O-methyltransferase which is involved in catechol amine
degradation has a common polymorphism V158M that has been implicated in
cognitive differences with individuals with the M allele performing
better. The M allele results in a less active enzyme and concomitantly
greater dopamine concentrations, suggesting that its effects are a
consequence of dopamine concentration. However, its effect might be
relatively subtle. The M/M homozygotes are relative rare globally
compared to V/V homozygotes or M/V heterozygotes. However, there is some
population differentiation of the variants with the maximum M/M presence
in White population and relatively low presence in Chinese/Japanese and
intermediate presence in Sub-Saharan Africans.

Further, quantitative trait associations studies have found linkages
between IQ and two chromosomal regions namely 2q24.1-31.1 and
6p25.3-22.3. These respectively overlap with chromosomal regions linked
with autism and reading disability/dyslexia. Several genes lie in these
chromosomal locations that have emerged as potential candidates in other
cognitive disorders. E.g. : SLC25A12: autism; NR4A2, DTNBP1, KIF13A,
NQO2: schizophrenia; RANBP9: fragile X syndrome; BBS5: Bardet-Biedl
syndrome. There are also other uncharacterized candidates in these
regions like: KCNH7, Neuritin 1, HTR5A, HTR3A and HTR3B. Thus, there
several indications that other genes with a role in individual
differences in intelligence might be uncovered in the near future.

[![](https://i2.wp.com/bp1.blogger.com/_ZhvcTTaaD_4/Ry6u5rxjPNI/AAAAAAAAAQE/g4l8byp3Jpc/s320/neuron_intel.png)](http://bp1.blogger.com/_ZhvcTTaaD_4/Ry6u5rxjPNI/AAAAAAAAAQE/g4l8byp3Jpc/s1600-h/neuron_intel.png)

Location and function of the gene products of major genes implicated in
intelligence (Click to enlarge).

An examination of the major genes implicated in intelligence shows an
interesting pattern. 4 out of the 7 allelic variants with an effect on
intelligence/cognition affect the regulatory regions, most likely
transcription control sites in non-coding regions of the gene. The only
known LIMK1 allele with a cognitive effect seems to be a null allele
resulting in complete gene deletion. This suggests that a notable
fraction of the variation in the genes resulting in intelligence
differences is subtle and does not alter protein function. Instead, the
chief effects of the variants are predicted to be in changing
concentrations or amounts of proteins that are available in within the
neuron. Only in two cases (RIMS1 and COMT) do we actually observe a
change in protein functional properties, through alteration of the
coding region. In at least the latter case the net result seems to be a
change in dopamine concentration. These observations suggest that most
effects of genetic variation on intelligence can in part be relatively
simply modeled a changes in protein or neurotransmitter concentration
which either directly affect neuronal architecture (e.g. DTNBP1) or
amount of available neurotransmitter or its receptor. This also raises
the possibility of relatively easily phenocopying such alterations
through non-genetic, biochemical interventions. Thus, intelligence
altering drugs, which have been a fascination from the earliest days of
Hindu medicine might not be out of place.

Beyond these genes, a recent study (Cochran et al) on the possible
genetic determinants for high IQ in the Ashkenazi Jewish population
uncovered a series of potential candidate genes. So far there is no
evidence for these having a general role in intelligence in association
studies. However, as the authors suggest some of this may have a been
uniquely selected in a particular Jewish population: elsewhere their
negative fitness effects might have eliminated them from the population.
The best of these candidates include genes like torsin encoding an AAA+
ATPase, mutated in Torsion dystonia. The mutation appears to affect a
glutamate in the helical C-terminal module of the AAA+ domain and might
hence affect its target protein interaction, and there by affect protein
translocation. The other candidate is CYP11B1 gene which encodes a
steroid 11-beta hydroxylase, which is mutated in non-classical
congenital Adrenal Hyperplasia but its exact role if any in elevating IQ
is unclear. The DNA repair group including BRCA1, the Bloom’s syndrome
helicase and some others have also been implicated along with brain-size
related genes with related functions such as [ASPM and
MCPH1](http://manollasa.blogspot.com/2005/09/evolving-brain.html).
However, at least in the case of the latter two no direct association
with IQ was found. However, ASPM and MCPH1 have been linked to the
emergence of tonal languages and alphabetic writing, suggesting that in
combination with some of the established IQ-affecting genes these and
other DNA repair genes could impact cognitive capacity in facets other
than that measure by proxies for g.

The case of FADS2:  It has been known from earlier studies that
breast-feed infants develop significantly higher IQ than non-breastfed
infants. However, this increase in IQ due to breast-feeding is dependent
on the presence of a particular allelic variant (SNP: rs174575) in the
FADS2 gene. This polymorphism is close to a predicted sterol response
element and might affect transcription factor binding. Individuals with
the more prevalent FADS2 allele in the tested population (mainly White
Anglo-Saxon) responded dramatically breast-feeding in terms of IQ,
whereas those with the less prevalent allele showed neither IQ elevation
nor depression in response breast-feeding. FADS2 encodes a membrane
associated delta-6 fatty acid desaturase with a N-terminal cytochrome b5
domain and a C-terminal multi-TM desaturase domain that is involved in
the synthesis of highly unsaturated fatty acids. It is possible that the
activity of this enzyme on precursors derived from breast-milk are
critical for generation of specific highly unsaturated fats accumulated
in the brain. An earlier association study had linked the intronic SNP
rs498793 in the same gene (different from the above one) with
attention-deficit/hyperactivity disorder. This SNP shows dramatic
difference between Chinese/Japanese one hand and Africans/Whites on the
other. It would be interesting to see if this allele might account for
behavioral differences between these ethnic groups, and whether it
interacts with other alleles in the same locus. Here again the
polymorphisms linked to phenotypic differences are merely involved in
regulatory effects rather than changing protein biochemistry.

The case of FADS2 again raises the possibility that dietary supplements
of particular fatty acids might be beneficial for development of
intelligence. More generally, it shows how environmental factors
themselves might uncover a further set of genes that affect development
of intelligence.

This finally leads to the politically contentious issue of intelligence
difference between different populations, ethnic groups or races.
Psychometry has consistently suggested differences in average g as
measured by IQ between different populations. The above survey of
genetic variation implicated in individual intelligence differences also
show marked population differences in their frequencies. This is
consistent with a genetic basis of the inter-population differences in
intelligence. However, we must keep in mind that we have not exhausted
all the genes involved in intelligence differences, nor have exhausted
the effects of all possible allelic variants found in different
populations. So we cannot yet reconstruct purely ground up a measure of
the amount of inter-population intelligence difference using the current
molecular data.
