+++
title = "Skin color of Indians (andxa0others)"

+++
Indians more than any other ethnic group are aware of the dramatic
variation in their skin color. This may be seen right in the same family
(especially in brahmin households from the southern part of the
peninsula) with all the shades ranging from fair as Western Eurasians to
dark as sub-Saharan Africans. What plays out in a family also plays out
over the entire country. There are some well-known trends: 1) The
gradient from north to south of the sub-continent with the extreme
Northerners close to European complexions and extreme Southerners more
closer to sub-Saharan Africans. 2) A less visible trend that is still in
need of more detailed study is the gradation between the varNa and
avarNa populations, with the latter tending to have more dark
individuals and possibly being darker on an average and than the former.

[![](https://i1.wp.com/bp3.blogger.com/_ZhvcTTaaD_4/RuYpT_XcXoI/AAAAAAAAAOU/IarDdpXSHqI/s320/skin_color_distribution.gif)](http://bp3.blogger.com/_ZhvcTTaaD_4/RuYpT_XcXoI/AAAAAAAAAOU/IarDdpXSHqI/s1600-h/skin_color_distribution.gif)

World skin color distribution of native populations (I believe this is
not entirely accurate but a an approximation that serves for
illustrative purposes).

In a sense India reflects the variation that is seen throughout the
world all within a subcontinent. Now on the global scale the populations
closer to the equator tend to be dark, while those from higher latitudes
are fair. In Eurasia, two otherwise only distantly related populations
are characterized by fair skin: the Europeans, and the Northern East
Asians like Chinese, Japanese and Koreans. All these have for long
suggested a major selective role played by solar radiation damage and
vitamin D production in shaping the biogeography of skin. The quest to
understand the mysteries of Indian skin color genetics began with
Haldane’s shot at the dark in the form of a 4 gene model to explain the
majority of skin color diversity in India. But we have now come close to
a real understanding with several new papers supplying nuggets of useful
data on genetics of skin color. I paste some of those references below
and provide a synthetic account with some related observations of mine.

[![](https://i0.wp.com/bp3.blogger.com/_ZhvcTTaaD_4/RuYpT_XcXnI/AAAAAAAAAOM/bds9pArFBY8/s320/mouse_underwhite.jpg)](http://bp3.blogger.com/_ZhvcTTaaD_4/RuYpT_XcXnI/AAAAAAAAAOM/bds9pArFBY8/s1600-h/mouse_underwhite.jpg)[![](https://i2.wp.com/bp2.blogger.com/_ZhvcTTaaD_4/RuYpTvXcXmI/AAAAAAAAAOE/A91uI81c1Z8/s320/golden_zfish.jpg)](http://bp2.blogger.com/_ZhvcTTaaD_4/RuYpTvXcXmI/AAAAAAAAAOE/A91uI81c1Z8/s1600-h/golden_zfish.jpg)

The mouse below is the Underwhite mutant (SLC45A2 gene). The fish on top
is the zebrafish golden mutant. The human on the left is the
corresponding human natural variant (both in SLC24A5 gene).

There are only two major biological purveyors of skin color shared by
all humans– melanin the skin pigment and hemoglobin the blood pigment. A
combination of the reflectivities of these two compounds colors skin. In
East Asians, additionally the layer of fat below the skin may provide an
additional yellowish tinge. Melanin is an ancient pigment in vertebrates
produced by melanocytes from the amino acid tyrosine (and cysteine in
the case of the brown pigments) in the derived lysosomes termed
melanosomes. Melanosomes are transported along the dendritic projections
of the melanocytes (which in someways resemble neural cells being
derived as sister cells in the neural crest) to neighboring
keratinocytes in skin, which are directly responsible for skin and hair
color. Earlier studies on mouse pigmentation identified over 100 genes
several orthologs of which are also mutated in human pigmentation
disorders. More recently the zebrafish golden gene was sequenced and its
human ortholog was found to show an allele that in humans shows a
mutation that correlates with lighter skin color. Thus a common pathway
of melanocyte function and skin pigmentation appears to function
throughout vertebrates.

The main results of the studies on Indian skin color genetics are thus:

\-3 genes account for a major component of color variation amongst
Indians:  
SLC24A5 (ortholog of zebrafish golden), TYR (ortholog of mouse Albino
gene) and SLC45A2 (ortholog of mouse Underwhite).

\-These genes are also been implicated in color variation between
Europeans and Africans. Light skin in Europeans and dark skin in
Africans is due to alternate alleles in the same genes. At least some of
the same alleles responsible for this color difference between Europeans
and Africans seem to be involved in Indians. The SLC45A2 shows an
allelic gradient from north to south in Europe. It might similarly be
different in distribution between north and south Indians, with decrease
in frequency in South India.

\-While the above allelic variation responsible for skin color variation
in Indians is as seen in Europeans and Central Asians it does not match
the situation in Asians (people from East like Chinese Japanese Koreans
etc). Here different set of genes appear to have convergently given rise
to light skin color.

The region around the SLC24A5 gene shows a considerable reduction of
heterozygosity suggesting a major selective sweep of this light skinned
allele through the population. Interestingly, it appears that the
light-skin in Indians and Central/Western Eurasians had a common origin
independent of those in Asians, especially in terms of the contribution
of the SLC24A5 allele. In a general sense this is not very surprising
given that the Indian varNa populations show a closer relationship and
Central/Western Eurasian populations to the exclusion of Asians. But
what does this mean? After all several Indians who have the SLC24A5
A111T “light allele” (SNP: rs1426654) are not exactly in sun-starved
climes where low vitamin D could trigger a major selective sweep. In
fact it is found even in Shri-Lankans in fairly significant frequency.
This might suggest that indeed it is ancestry rather than adaptation
that has resulted in deep South Indian possessing this SLC24A5 allele in
the observed frequencies. It means that Indians, including many from
South India and Shri Lanka might have had ancestors from Northern
latitudes. The results from these studies also suggest that in South
India and Lanka there might actually be selection by UV-damage against
the “light alleles” like that of SLC45A2 keeping its frequency low
(Given that it is much lower than expected when compared to SLC24A5. In
fact Soejima et al propose that it might be a useful forensic marker for
Lankans and Tamils). Yet the retention of certain percentage of light
alleles in tropical India might also suggest a sexual selection for such
alleles.

So what do these genes do?

TYR encodes the tyrosinase which catalyze the oxidation of phenols to
quinones in the first step of melanin biosynthesis. So evident the
allele in Indian populations directly influence the amount or efficiency
of the enzyme at first step. SLC24A5 and the SLC45A2 genes are more
mysterious transporter proteins. SLC24A5 protein is a cation antiporter
that is likely to exchange 4 Na+ for Ca+ and K+. The SLC45A2 protein is
a member of the major facilitator superfamily of transporters that
performs symport of sugar and sugar derivatives with Na+. This suggests
that the SLC24A5 and SLC45A2 are functionally linked via the transport
of Na+. Workers have suggested that the ion transport by these proteins
may be related to efficiency of melanosome function by affecting pH
(which is different in white and black skin) and providing calcium for
proteolysis of the SILV protein.

However, I suspect that they are ignoring the issue of sugar transport
by SLC45A2. The SLC24A5 A111T mutation maps to a loop between the TM
regions of the transporter and the ancestral A in this position is
highly conserved, and is seen in orthologous proteins in plants. Thus
this state was present common ancestor of plants and animals, suggesting
a key functional role in formation of a structure in the soluble part of
the transporter. Hence, the A111T mutation probably seriously
compromises function of this promoter. Likewise the SLC45A2 L374F
mutation is in a soluble loop and the position is alway occupied by an
aliphatic side chain residue in orthologous proteins, but never
aromatic. Thus, again the L374F probably seriously affect the effective
functioning of the transporter. We wonder, given that the melanosome is
a derived lysosome, if this transport system ultimately affects the
transport of a sugar. The transport of this sugar might be critical for
processing of glycosylated proteins in the melanosome and there by their
appropriate localization. Future investigation of this angle might be
appropriate.

The key papers:  
Norton HL, Kittles RA, Parra E, McKeigue P, Mao X, Cheng K, Canfield
VA,  
Bradley DG, McEvoy B, Shriver MD.  
Genetic evidence for the convergent evolution of light skin in Europeans
and East Asians.  
Mol Biol Evol. 2007 Mar;24(3):710-22.

Soejima M, Koda Y.  
Population differences of two coding SNPs in pigmentation-related genes
SLC24A5 and SLC45A2.  
Int J Legal Med. 2007 Jan;121(1):36-9

A genome-wide association study of skin pigmentation in a South Asian
population  
Renee P. Stokowski, P.V. Krishna Pant, Tony Dadd, Amelia Fereday, David
A. Hinds, Carl Jarman, Wendy Filsell, Rebecca S. Ginger, Martin R.
Green, Frans J. van der Ouderaa, David R. Cox  
The American Journal of Human Genetics in press
